test/integration/discovered_hosts_test.rb

```diff
@@ -9,6 +9,7 @@ class DiscoveredHostsTest < IntegrationTestWithJavascript
     assert discovered_notification_blueprint
     discovered_host.save!
     visit discovered_hosts_path
+    page.evaluate_script("document.getElementById('fixedPropertiesSelector-#{discovered_host.id}').classList.remove('fade')")
```

Comment:
The modal should not be fading as sometimes the click event misses the button as it is still moving.